src/DirectApiService.php

```diff
@@ -162,7 +162,7 @@ class DirectApiService
     private $useSandbox = false;
 
     public function __construct(
-        string $token,
+        string $token = null,
```

Comment:
Вероятно стоит пометить `$token` как `nullable`
```suggestion
        ?string $token = null,
```